### ISSUE TITLE
[css-subtree-visibility-1] which tree is this looking at subtrees of?

### ISSUE THREAD

**Issue by dbaron** [WICG/display-locking#128]
@kenchris and I are looking at [the CSS Subtree Visibility draft](https://wicg.github.io/display-locking/) as part of w3ctag/design-reviews#306.

It makes a number of references to "subtree" but isn't clear about what trees it's talking about:  is this the initial DOM tree (hopefully not), the flattened tree (maybe) (not sure if "flattened tree" is a current technical term, though), or the CSS box tree?

**Comment by rakina** [comment]
I think we intend on using the flattened tree as discussed here: https://github.com/whatwg/html/pull/4862/files#r318853795, with the definition of "flattened element tree" specced in CSS Scoping. Will that be ok?

**Comment by vmpstr** [comment]
Also please see #32 for a supporting example that we want flat tree here.

**Comment by chrishtr** [comment]
Suggested text for edit to draft spec should I think be something like:

Define "subtree" as all boxes and text runs generated by nodes that are descendants of a given element in the flat tree (flat tree defined [1]).

Then use "subtree" throughout.

[1](https://drafts.csswg.org/css-scoping/#flattening)

**Comment by chrishtr** [comment]
@tabatkins

**Comment by tabatkins** [comment]
Yeah, for DOM purposes this is the descendants in the flattened element tree; for most CSS purposes it'll be the boxes generated by those descendants.
